webpack/scenes/ContentViews/ContentViewPage.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from 'react';
+import ContentViewTable from './Table/ContentViewTable';
+
+const contentViewIndexFakeData = require('./data/index');
+// Uncomment and comment out above to see empty state
+// const contentViewIndexFakeData = { results: [] };
+
+const ContentViewPage = () => {
+  const [contentViews, setContentViews] = useState(null);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setContentViews(contentViewIndexFakeData);
+    }, 5000);
```

Comment:
added a timeout to show the loading spinner :cyclone:

Reply:
This is where the redux function that calls the Content View index endpoint will go